_pages/openid-connect.md

```diff
@@ -271,3 +340,32 @@ login.gov supports some of the [standard claims from OpenID Connect 1.0][standar
 
  * **sub**
    Unique ID for this user. This ID is unique per client.
+
+<script type="text/javascript">
```

Comment:
This is simple enough, I think it's easier than adding a dependency on jQuery, unless you feel strongly? `document.querySelectorAll`, a new standard, is inspired by jQuery?